backend/db/migrate/20221122091158_create_books.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :books do |t|
```

Comment:
I would suggest that you use UUID for all tables. Especially for tables that can be visible in the frontend.